src/contracts/identity.ts

```diff
@@ -130,3 +154,15 @@ export const isAuthorizedTo = async (
 
   return Identity__factory.connect(contractAddress, provider).isAuthorizedTo(address, permission, blockNumber);
 };
+
+const getBeaconAddress = async () => {
+  const {
+    provider,
+    contracts: { beacon },
+  } = getConfig();
```

Comment:
config option?

Reply:
This follows the pattern of other contract retrieval functions in `identity.ts` that expect the config to already have been set. I'd rather not expose it here because I'd then have the same issue for `createAndRegisterBeaconProxy`, and so on.